plugins/@grouparoo/clickhouse/public/@grouparoo/clickhouse/clickhouse.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you only need the `.png` - remove the other version of the logo if you aren't using it

Reply:
I opted to use the svg logo instead. In general, it's better to use the SVG version over the png because it renders a lot more sharper for different screen sizes and display pixel densities.